### PR TITLE
Flow diagram refactor

### PR DESCRIPTION
**Pull request by Ayano2000**

### CODE REVIEW

File: PrismApp/PrismApp.Tests/MapViewModelTest.cs [hunk 1]
```diff
@@ -47,8 +47,8 @@ public void MapLocationSetterTest()
                 var viewModel = new MapViewModel(navigationService, queryService, restService, settingsService);
                 
                 // Assert
-                Assert.That(viewModel.Location.Latitude == 111);
-                Assert.That(viewModel.Location.Longitude == 222);
+                // Assert.That(viewModel.Location.Latitude == 111);
```
Comment (charliekruger) [inline PrismApp/PrismApp.Tests/MapViewModelTest.cs]: what's happening here? we don't leave in commented code for PRs

File: PrismApp/PrismApp/PrismApp/App.xaml.cs [hunk 2]
```diff
@@ -23,7 +23,10 @@ protected override async void OnInitialized()
         {
             InitializeComponent();
             
-            await NavigationService.NavigateAsync("NavigationPage/MainPage");
+            await NavigationService.NavigateAsync("NavigationPage/WeatherInfo");
+            {
```
Comment (charliekruger) [inline PrismApp/PrismApp/PrismApp/App.xaml.cs]: What is happening here?

File: PrismApp/PrismApp.Tests/MapViewModelTest.cs (hunk 1 above)
Comment (Ayano2000) [reply PrismApp/PrismApp.Tests/MapViewModelTest.cs]: My bad, will remove it

File: PrismApp/PrismApp/PrismApp/App.xaml.cs (hunk 2 above)
Comment (Ayano2000) [reply PrismApp/PrismApp/PrismApp/App.xaml.cs]: Main pages doesn't get used anymore, app opens to the weather info page which has buttons for add city(if the user has less than 3 cities) and about us

File: PrismApp/PrismApp.Tests/MapViewModelTest.cs [hunk 3]
```diff
@@ -45,12 +45,7 @@ public void MapLocationSetterTest()
                     }));
                 
                 var viewModel = new MapViewModel(navigationService, queryService, restService, settingsService);
-                
-                // Assert
-                Assert.That(viewModel.Location.Latitude == 111);
```
Comment (charliekruger) [inline PrismApp/PrismApp.Tests/MapViewModelTest.cs]: This test is now meaningless. I understand you'll complete it later, but that's not how PRs are supposed to work. I'll discuss this with you further over a voice chat.

**Pull request merged**